packages/pages-shared/environment-polyfills/miniflare.ts

```diff
@@ -2,11 +2,13 @@ import { polyfill } from ".";
 
 export default async () => {
 	const mf = await import("@miniflare/core");
+	const { HTMLRewriter } = await import("@miniflare/html-rewriter");
```

Comment:
We aren't passing `caches`/`waitUntil` to the `generateHandler()` when we actually use this in `wrangler pages dev` (https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/miniflare-cli/assets.ts#L170-L237) so why have you added `HTMLRewriter` here?

Reply:
I should revert this. I was originally trying to use this function directly in the jest setup file.

Reply:
In [packages/pages-shared/environment-polyfills/types.ts](https://github.com/cloudflare/wrangler2/blob/main/packages/pages-shared/environment-polyfills/types.ts#L11) we declare it as a global that's available and we use it freely. The test code passes in `caches` and `waitUntil` so they can be spied upon

Reply:
Ah, fair enough then. I'd be tempted to pull it out of global and pass it through like the rest of these things — I must have just overlooked it before — but no biggie and adding it here is fine :)